bin/parse.rb

```diff
@@ -0,0 +1,33 @@
+#!/bin/env ruby
+
+root_path = File.expand_path('..', File.dirname(__FILE__))
+$LOAD_PATH.unshift(root_path) unless $LOAD_PATH.include?(root_path)
+Dir.chdir root_path
+
+def nickParse aliases
+  #File.open 'data/tmp.log', 'w'
+  aliases.each {|a|
+    #File.open 'log' 'r'
+    p "alias: #{a}"
+  }
+#	for n in "${nicks[@]}"; do
```

Comment:
once this script is functionally equal to parse.sh, i'll make sure there's no commented code